### ISSUE TITLE
filter issue

### ISSUE THREAD

**Issue by edi** [LeaVerou/prefixfree#206]
filter attribute is not getting mutated at all. I am using in css filter: blur(5px); and the browser (chrome latest version) fails to parse , since it can read only prefixed -webkit-filter: blur(5px), which works if I write it this way.

**Comment by felipeduardo** [comment]
"filter: invert()" not also works

**Comment by edi** [comment]
@felipeduardo , filter property does not work at all.

**Comment by LeaVerou** [comment]
Yup, that’s a known issue and happens with all the SVG properties that got new syntax and got adopted into CSS. The reason is that PF sees that they are available without a prefix too, so it thinks they don’t need prefixing.
We could just prefix them blindly, as long as the prefixed version is available, but I’m worried that once browsers start unprefixing them, we will end up using the worse implementation of the two. Also, it might break uses of them in actual SVG, but I haven't tested that.
Ideally, PF should look at the entire definition and check if that’s valid without a prefix, but I’m worried about performance.
In any case, I consider this one of the of the biggest PF issues currently and I really want to find a way to solve it. It bugs me a lot too. :(

**Comment by nick4fake** [comment]
Prefixfree is useless due to this bug, as we need prefixer on server-side because of this.

**Comment by edi** [comment]
Not really useless, I use it on almost every project of mine, and where it comes to filter and stuff I just manually insert the prefixes.
